include/constants.py

```diff
@@ -119,6 +119,182 @@
+                                                '10': 32968.80959}},
+                            'Digital_V2':{'32':{'100':4466.744263,
+                                                '90': 4053.765767,
+                                                '80': 3744.031895, 
+                                                '70': 3434.298023, 
+                                                '60': 3124.564151, 
+                                                '50': 2814.830279, 
+                                                '40': 2401.851783, 
+                                                '30': 2092.117911, 
+                                                '20': 1782.384039, 
+                                                '10': 1472.650167},  
+                                          '64':{'100':17654.27322,
+                                                '90': 16216.06481,  
+                                                '80': 14983.31474, 
+                                                '70': 13545.10633, 
+                                                '60': 12312.35626, 
+                                                '50': 11079.6062, 
+                                                '40': 9641.397787, 
+                                                '30': 8408.647721, 
+                                                '20': 6970.439311, 
+                                                '10': 5737.689245},
+                                        '128':{'100': 70237.24474,
+                                                '90': 64904.19392,  
+                                                '80': 59571.14309, 
+                                                '70': 54238.09226, 
+                                                '60': 48905.04144, 
+                                                '50': 43982.22529, 
+                                                '40': 38649.17446, 
+                                                '30': 33316.12363, 
+                                                '20': 27983.07281, 
+                                                '10': 22650.02198},
+                                        '256':{'100': 280471.5471,
+                                                '90': 259128.5,  
+                                                '80': 237785.453, 
+                                                '70': 217263.2925, 
+                                                '60': 195920.2454, 
+                                                '50': 175398.0849, 
+                                                '40': 154055.0379, 
+                                                '30': 132711.9909, 
+                                                '20': 112189.8303, 
+                                                '10': 90846.78326}}}
+Digital_xbar_pow_leak_dict = {  '32' :5.575928889,          #in mW 
+                                '64' :12.82466678,
+                                '128':40.24037556,
```

Comment:
Would it be better to add dictionaries asscoaited to digital MVMU in a separate `constants-digital-mvmu.py` file for readability ?